styles/style.css

```diff
@@ -0,0 +1,96 @@
+body {
+  margin: 0;
+  padding: 0;
+  width: 100vw;
+  overflow: hidden;
+}
+
+/* Start of header styles */
+header {
+  height: 72px;
+  color: white;
+  display: flex;
+  align-items: center;
+  padding: 0 20px;
+  font-size: 18px;
+  font-family: Inter, sans-serif;
+  justify-content: space-between;
+  background-color: #1c1a19;
+}
+
+#bars > svg {
+  color: white;
+  width: 25px;
+  height: 25px;
+}
+
+#bars > svg:hover {
+  color: yellow;
+}
+
+/* Start of headline-section styles */
+.headline-section {
+  width: 100%;
+  height: 100vh;
+  margin-top: -20px;
+  overflow: hidden;
+  display: flex;
+  background-image: url("/public/images/headline-section.png");
```

Comment:
I think you use template-2. 

- [ ] You should stick to design.  [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%20font%2C%20colors%2C%20images%2C%20tests%2C%20margins)%20using%20the%20templates%20in%20Figma.).

- [ ] You should same layout(icons), same background image, same font family. Please change it.

**Original:**
![image](https://user-images.githubusercontent.com/38635158/129082117-d216a0ed-a4b3-41c0-a5fc-8e82b3a05344.png)
____
**Yours:**
![Screenshot from 2021-08-11 21-22-03](https://user-images.githubusercontent.com/38635158/129082673-36c51a08-941c-42b5-9fa0-39a5bc9cb2ed.png)

Reply:
Well done

Reply:
Thank you Emmyn5600